tools/dockerfile/grpc_artifact_protoc_aarch64/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 # On aarch64 the protoc and codegen plugins are built via crosscompilation.
-FROM dockcross/linux-arm64
+FROM dockcross/manylinux2014-aarch64
```

Comment:
In the Dockerfile `grpc_artifact_python_manylinux2014_aarch64/Dockerfile` the image used is a tagged version:
`FROM dockcross/manylinux2014-aarch64:20230905-7b2d74f`
We might need to do that here to be consistent.

Reply:
Done!